draft-thoji-scone-protocol.md

```diff
@@ -57,24 +57,34 @@ informative:
 --- abstract
 
 On-path network elements can sometimes be configured to apply rate limits to
```

Comment:
```suggestion
This document describes a protocol where on-path network elements
can give endpoints their perspective on what the maximum achievable
throughput might be for QUIC flows.
```

This can be shorter, and I think we don't need to get into what might cause a network element to offer this advice, rather we just need to establish that network elements have a perspective and that that perspective might be useful to communicate.